src/main/java/uk/gov/hmcts/cmc/claimstore/models/Claim.java

```diff
@@ -116,6 +124,14 @@ public LocalDateTime getRespondedAt() {
         return respondedAt;
     }
 
+    public Optional<ResponseData> getResponse() {
+        return Optional.ofNullable(response);
+    }
+
+    public String getDefendantEmail() {
```

Comment:
This is obligatory field. Don't we use Optional<> then?

Reply:
It should be, but countyCourtJudgment doesn't appear to be ;)